server/app/services/geo/AddressSuggestion.java

```diff
@@ -0,0 +1,45 @@
+package services.geo;
+
+import com.google.auto.value.AutoValue;
+import services.Address;
+
+/**
+ * Represents an address candidate in the context of address candidates
+ *
+ * <p>See {@link AddressCandidates} for details on address candidates
+ */
```

Comment:
suggestion (blocking): javadoc needs updating. Please describe what an address suggestion is, its role in the system, etc.